source/developer/webhooks-incoming.md

```diff
@@ -33,6 +33,8 @@ Build Break - Project X - December 12, 2015 - 15:32 GMT +0
 
 ---
 
+Note: If the contents of incoming webhooks are cluttering your debug log, disable it in the `config/config.json` by setting `"EnableWebhookDebugging": false`
```

Comment:
Thanks @apheleia! I think the placement of this note might take a little too high priority in the document. I'd propose removing this note given that you've added similar info to the "Additional Notes" section below?